nautobot/dcim/filters.py

```diff
@@ -1556,15 +1572,12 @@ class Meta:
             "label",
             "length",
             "length_unit",
-            "termination_a_id",
-            "termination_b_id",
         ]
 
-    def filter_device(self, queryset, name, value):
-        queryset = queryset.filter(
-            Q(**{"_termination_a_{}__in".format(name): value}) | Q(**{"_termination_b_{}__in".format(name): value})
-        )
-        return queryset
+    def filter_by_termination(self, queryset, name, value):
+        # Filter by a related object cached on CableTermination. Note the underscore preceding the field name.
+        # Supported objects: device, rack, location, site
+        return queryset.filter(**{f'terminations___{name}__in': value}).distinct()
```

Comment:
This doesn't need to be a method filter. This could be done with a regular `MultiValue{Foo}Filter` using `field_name="terminations__{existing_field_name}", lookup_expr="in", distinct=True` passed through on each filter definition.

Because of the method code, the `field_name` being defined is not even being used here, and only the `name` of the assigned filter is being passed into the filter (e.g. `rack` is used not `rack__name` in that case).

e.g.
```python
    rack_id = MultiValueUUIDFilter(field_name="terminations___rack_id", lookup_expr="in", distinct=True, label="Rack (ID)")
```

Admittedly I don't understand this comment of the "underscore preceding the field". Does that mean each of those is using the `name` value that comes in from the field prefixed with an underscore e.g. `_rack_id`, `_rack`, `_site_id`, etc.?